MapboxNavigation/CarPlayManager.swift

```diff
@@ -82,6 +82,17 @@ public protocol CarPlayManagerDelegate {
      * Called when navigation ends so that the containing app can update accordingly.
      */
     @objc func carPlayManagerDidEndNavigation(_ carPlayManager: CarPlayManager) -> ()
+    
+    /**
+     Called when the carplay manager will disable the idle timer.
+     
+     Implementing this method will allow developers to change whether idle timer is disabled when carplay is connected and the vice-versa when disconnected.
+     
+     - parameter carPlayManager: The carplay manager that will change the state of idle timer.
+     - returns: A bool indicating whether to disable idle timer when carplay is connected and enable when disconnected.
+     */
+    @objc(carplayManagerShouldDisableIdleTimer:)
```

Comment:
nit: this automatically bridges to the same signature, @objc would be sufficient.